webapp/common.js

```diff
@@ -64,7 +64,9 @@ function setWindow(window, back) {
 
     const backButton = $('#back-button');
     backButton.off();
+    backButton.hide();
     if (back) {
+        backButton.show();
```

Comment:
I'm not sure whether this does what you want. I think you just want to hide the back button in the `else` clause in case `history.length == 1`.

Reply:
agreed